sdk/eventhubs/azure_messaging_eventhubs/src/common/management.rs

```diff
@@ -127,7 +127,6 @@ impl ManagementInstance {
         let name: String = response
             .get(EVENTHUB_PROPERTY_NAME)
             .ok_or_else(|| EventHubsError::from(ErrorKind::InvalidManagementResponse))?
-            .clone()
```

Comment:
I remember seeing a bunch more of these in another file. Might've been tests, though, so not a big issue. But might take a look.